src/include/mpir_hw_topo.h

```diff
@@ -0,0 +1,152 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/*
+ *  (C) 2019 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ *
+ */
+
+#ifndef MPIR_HW_TOPO_H_INCLUDED
+#define MPIR_HW_TOPO_H_INCLUDED
+
+/*
+ * Node hardware object types
+ */
+typedef enum {
+    MPIR_NODE_HW_OBJ__MACHINE = 0,
+    MPIR_NODE_HW_OBJ__PACKAGE = 1,
+    MPIR_NODE_HW_OBJ__CORE = 2,
+    MPIR_NODE_HW_OBJ__PU = 3,
+    MPIR_NODE_HW_OBJ__L1CACHE = 4,
+    MPIR_NODE_HW_OBJ__L2CACHE = 5,
+    MPIR_NODE_HW_OBJ__L3CACHE = 6,
+    MPIR_NODE_HW_OBJ__L4CACHE = 7,
+    MPIR_NODE_HW_OBJ__L5CACHE = 8,
+    MPIR_NODE_HW_OBJ__L1ICACHE = 9,
+    MPIR_NODE_HW_OBJ__L2ICACHE = 10,
+    MPIR_NODE_HW_OBJ__L3ICACHE = 11,
+    MPIR_NODE_HW_OBJ__GROUP = 12,
+    MPIR_NODE_HW_OBJ__NUMANODE = 13,
+    MPIR_NODE_HW_OBJ__BRIDGE = 14,
+    MPIR_NODE_HW_OBJ__PCI_DEVICE = 15,
+    MPIR_NODE_HW_OBJ__OS_DEVICE = 16,
+    MPIR_NODE_HW_OBJ__MISC = 17,
+    MPIR_NODE_HW_OBJ__GPU_DEVICE = 18,
+    MPIR_NODE_HW_OBJ__NETWORK_DEVICE = 19,
+    MPIR_NODE_HW_OBJ__DDR = 20,
+    MPIR_NODE_HW_OBJ__HBM = 21,
+} MPIR_Node_hw_obj_type;
```

Comment:
I think the objects should be `MPIR_NODE_HW_OBJ_TYPE__*` to be consistent with the name of the enum.

Reply:
Sorry my mistake